onnxruntime/core/providers/webnn/builders/helper.cc

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
If this file is created from us, we probably should use Intel copyright. If this file is derived from another file, we should keep the original copy right and append Intel one.